code/parachain/runtime/picasso/src/migrations.rs

```diff
@@ -57,167 +69,55 @@ impl OnRuntimeUpgrade for TechCollectiveRenameMigration {
 	}
 }
 
-pub mod migrate_oracle {
+pub mod migrate_gov {
 	use super::*;
 	use frame_support::traits::{GetStorageVersion, StorageVersion};
 
-	use sp_std::vec::Vec;
-
-	pub struct MigrateOracle;
-
-	const ORACLE_V2: StorageVersion = StorageVersion::new(2);
-
-	/// change some of the pools' lp token ids
-	fn migrate_oracle() -> Weight {
-		let mut answers_accounts: Vec<_> = Vec::new();
-		for account in oracle::pallet::AnswerInTransit::<Runtime>::iter_keys() {
-			answers_accounts.push(account);
-		}
-
-		let mut preprices_assets: Vec<_> = Vec::new();
-		for asset in oracle::pallet::PrePrices::<Runtime>::iter_keys() {
-			preprices_assets.push(asset);
+	use frame_support::traits::LockableCurrency;
+	use hex_literal::hex;
+	use sp_runtime::AccountId32;
+	pub struct MigrateGov;
+
+	const REFERENDA_V2: StorageVersion = StorageVersion::new(2);
+	const DEMOCRACY_ID: LockIdentifier = *b"democrac";
+
+	fn migrate() -> Weight {
+		// set key for relayer committee
+		let relayer_address = sp_runtime::MultiAddress::Id(AccountId32::from(hex!(
+			"868232e15789eaae263d655db7d222fcf5ffa5f6f8da4e46d32609312fcf6e60"
+		)));
+		membership::pallet::Pallet::<Runtime, NativeRelayerMembership>::add_member(
+			frame_system::RawOrigin::Root.into(),
+			relayer_address.clone(),
+		)
+		.expect("should add");
+		membership::pallet::Pallet::<Runtime, NativeTechnicalMembership>::remove_member(
+			frame_system::RawOrigin::Root.into(),
+			relayer_address,
+		)
+		.expect("should remove");
```

Comment:
Why do we need to add and remove?

Reply:
i move hyperspace to a separate comittee. i do it in migration so hyperspace doesnt stop